docs/sources/http_api/short_url.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Short URL HTTP API "
+description = "Grafana Short URL HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "shortUrl"]
+aliases = ["/docs/grafana/latest/http_api/short_url/"]
+type = "docs"
+[menu.docs]
+name = "Short URL"
+parent = "http_api"
++++
+
+# Short URL API
+
+This API can be used to create shortened URLs. This allows URLs containing complex query parameters to be represented by smaller, simpler URLs of the format `/goto/:uid`.
```

Comment:
Use this API to create shortened URLs. A short URL represents a longer URL containing complex query parameters in a smaller and simpler format.